infra/cmake/packages/TensorFlowSource-2.7.0/TensorFlowSourceConfig.cmake

```
@@ -0,0 +1,18 @@
+function(_TensorFlowSource_import)
+  if(NOT DOWNLOAD_TENSORFLOW)
+    set(TensorFlowSource_FOUND FALSE PARENT_SCOPE)
+    return()
+  endif(NOT DOWNLOAD_TENSORFLOW)
+
+  nnas_include(ExternalSourceTools)
+  nnas_include(OptionTools)
+
+  envoption(TENSORFLOW_2_7_0_URL https://github.com/tensorflow/tensorflow/archive/v2.7.0.tar.gz)
```

Same with `gemmlowp`

```suggestion
  envoption(EXTERNAL_DOWNLOAD_SERVER "https://github.com")
  envoption(TENSORFLOW_2_7_0_URL ${EXTERNAL_DOWNLOAD_SERVER}/tensorflow/tensorflow/archive/v2.7.0.tar.gz)
```

---

I deleted this file